src/main/java/org/jpeek/XslReport.java

```diff
@@ -109,10 +110,11 @@
      * Ctor.
      * @param xml Skeleton
      * @param name Name of the metric
+     * @param calc Calculus
      */
-    XslReport(final XML xml, final String name) {
+    XslReport(final XML xml, final String name, final Calculus calc) {
         this(
-            xml, name, new HashMap<>(0),
+            xml, name, new HashMap<>(0), calc,
```

Comment:
@HDouss Would be good to create another todo to have a constructor here without necessity to pass new `HashMap<>(0)`.

Reply:
@fanifieiev I don't understand. This constructor is taking no `params` arg. He's delegating to another constructor with a "default" params which is `new HashMap<>(0)`.  `params` is a final field of the class that should be initialized anyway (with a non-null value). Besides, this is not really relevant to this PR.

Reply:
@HDouss Sorry, I mean the delegated constructor, but now I see that. No need to change.
Regarding the "Besides, this is not really relevant to this PR": I do understand, that reviewers needs to ask to change or fix only the code the dev introduces, but there is a lot more code visible to reviewers, not only sth is changed by dev specifically and I strongly believe that REVs can at least ask dev to leave a todo for refactoring or fixing.

Reply:
@fanifieiev I mostly agree, but for this specific point, these constructors (caller and callee) existed for a long time now. In my opinion, it is more suitable to submit a new issue. Thank you anyway.